CMakeLists.txt

```diff
@@ -57,6 +57,16 @@ check_library_exists(m pow "" HAVE_LIBM)
 check_include_file("dirent.h" HAVE_DIRENT_H)
 check_symbol_exists(strcasecmp "strings.h" HAVE_DECL_STRCASECMP)
 check_symbol_exists(strncasecmp "strings.h" HAVE_DECL_STRNCASECMP)
+check_c_source_compiles("
+    #include <windows.h>
+    int main()
+    {
+        CreateWaitableTimerEx(NULL, NULL, CREATE_WAITABLE_TIMER_HIGH_RESOLUTION);
```

Comment:
Doesn't this take four arguments?

Reply:
Right, fixed here: https://github.com/fabiangreffrath/woof/pull/1849/commits/9c523b793118eb87b40c54934f6abe6a9cb678e8